plejd/PlejdApi.js

```diff
@@ -375,30 +375,46 @@ class PlejdApi {
 
           this.deviceRegistry.addOutputDevice(outputDevice);
         }
-      }
+      } else {
+        // The device does not have an output. It can be assumed to be a WPH-01 or a WRT-01
+        // Filter inputSettings for available buttons
+        const inputSettings = this.siteDetails.inputSettings.filter(
+          (x) => x.deviceId === device.deviceId
+            && (x.buttonType === 'DirectionUp'
+              || x.buttonType === 'DirectionDown'
+              || x.buttonType === 'RotateMesh'),
+        );
+
+        // For each found button, register the device as an inputDevice
+        inputSettings.forEach((input) => {
+          const bleInputAddress = this.siteDetails.deviceAddress[input.deviceId];
```

Comment:
Is this really correct (probably is since you tested it)? 

If it mimics output we should probably fetch BLE address from `inputAddress[deviceId][inputIndex]`

I looked quickly at the API response in my system. Interestingly `inputAddress[deviceId][0]` is NOT the same as `deviceAddress[deviceId]` (which is the case for outputs)...

So... When is `inputAddress` used if not here??

Reply:
inputAdress cannot be used for broadcasts like this as it changes with configuration.
Eg. Scene gives 255.
So this is correct I think.

Reply:
Ok, this requires a bit of an explanation. I think you missed the "trick" I made here.

The trick is that I merge the BLE address with the input and stores it to make it possible to differentiate the broadcasts from the different buttons.

BLE 11 becomes BLE 11_0, 11_1, 11_2 and 11_3.

I've improved the code now for better readability by adding and using this
```
  getUniqueBLEId(bleAdress, inputIndex) {
    return `${bleAdress}_${inputIndex}`;
  }
```

Reply:
Check the code that is coming (soon) when I update the PR. When we agree on this I will resolve the conversation